classes_part1.py

```diff
@@ -0,0 +1,61 @@
+import datetime
+import time
+import pet
+
+# Create empty array to store instances of pets in
+pet_list = []
+
+# Get info about Pet 1 and store it
+print("Let's start by gathering basic data about your first pet.\n")
+pet_1 = pet.pet('','','')
+pet_1.getName()
+pet_1.getCategory()
+pet_1.getSubtype()
+#   Add Pet 1 to array
+pet_list.append(pet_1)
+
+# Get info about Pet 2 and store it
+print("\nNow let's gather more detailed data about your second pet.\n")
+pet_2 = pet.pet('','','')
+pet_2.getName()
+pet_2.getCategory()
+pet_2.getSubtype()
+pet_2.getBirthdate()
+pet_2.getColors()
+pet_2.getHasTail()
+pet_2.getHasWings()
+pet_2.getNumOfLegs()
+pet_2.getLastFed()
+#   Add Pet 2 to array
+pet_list.append(pet_2)
+
+# Hard code info for Pet 3 and Pet 4
+print("\nI know you have also have 2 fish, Flotsam and Jetsam.  Let me enter the info I know about them now...")
+time.sleep(2)
+#   Add Pet 3 to array
+pet_3 = pet.pet('Flotsam', 'fish', 'goldfish', '', '', '', '', '', datetime.date.today() - datetime.timedelta(days=1), '')
```

Comment:
The correct way to do this is either use named parameters or only send in the first 3, then call a method like pet_3.setLastFed(date) passing in your equation for the date.  Don't just pass in '' for each value, as now they won't get their defaults of None and False (I don't think), because you've set them to a null string '' intead.